docs/releases/reason-codes.yml

```
@@ -65,3 +65,59 @@ evidence_check:
+MISSING_FIELD: "A required field is missing from the manifest."
+INVALID_ENUM: "An enum value is invalid."
+DRIFT_DETECTED: "Drift detected between generated and committed evidence."
+DRIFT_ERROR: "Error occurred during drift detection."
+SIG_ORPHAN: "Orphaned signature found without corresponding artifact."
+SIG_INVALID: "The signature is invalid or could not be verified."
+COSIGN_MISSING: "Cosign bundle is missing."
+SIG_MISSING: "Artifact signature is missing."
+ATTESTATION_MISSING: "Provenance attestation is missing."
+SHA256SUMS_INVALID_FORMAT: "The checksums file has an invalid format."
+INVALID_JSON: "The JSON content is invalid."
+SCHEMA_MAJOR_UNSUPPORTED: "The schema major version is not supported."
```

YAMLlint (1.38.0)

[error] duplication of key "SCHEMA_MAJOR_UNSUPPORTED" in mapping

(key-duplicates)